test/homepage-noweb-test.js

```diff
@@ -68,6 +69,25 @@ suite.addBatch({
                 assert.ifError(err);
                 br.assert.success();
                 br.assert.text("H1", "pump.io API");
+            },
+            "it not has login or register buttons": function(err, br) {
+                br.assert.success();
+                br.assert.evaluate('document.getElementById("login")', null);
+                br.assert.evaluate('document.getElementById("register")', null);
+            }
+        },
+        "and we request javascript file": {
+            topic: function() {
+                httputil.head("http://localhost:4815/javascript/pump.js", {
+                    "User-Agent": "Mozilla/5.0 (Windows NT 6.3; Trident/7.0; rv:11.0) like Gecko"
```

Comment:
We don't actually need this. The test you pulled this from was specifically testing some user-agent sniffing, but in general the user agent doesn't matter.

Reply:
Is't necesary put any user agent?

Reply:
Nope. Cause the affected code doesn't do anything special based on the user agent. So Node's default is fine.